Add note re:sharedWorker in browser devtools (#31734)
mdn/content: files/en-us/web/api/sharedworker/index.md

When the port is started, both scripts post messages to the worker and handle messages sent from it using `port.postMessage()` and `port.onmessage`, respectively:

> > **Note:** In order to debug your SharedWorker you need to navigate to the correct inspector. For example in Chrome web worker inspection can be opened from `chrome://inspect/#workers` and in FireFox the location is `about:debugging#workers`

Reviewer: ```suggestion
> **Note:** You can use browser devtools to debug your SharedWorker, by entering a URL in your browser address bar to access the devtools workers inspector; for example, in Chrome devtools, the URL [`chrome://inspect/#workers`](chrome://inspect/#workers), and in FireFox, the URL [`about:debugging#workers`](about:debugging#workers).
```

> > **Note:** You can use browser devtools to debug your SharedWorker, by entering a URL in your browser address bar to access the devtools workers inspector; for example, in Chrome devtools, the URL `chrome://inspect/#workers`, and in FireFox, the URL `about:debugging#workers`.

Reviewer: ```suggestion
> **Note:** You can use browser devtools to debug your SharedWorker, by entering a URL in your browser address bar to access the devtools workers inspector; for example, in Chrome, the URL `chrome://inspect/#workers`, and in FireFox, the URL `about:debugging#workers`.
```

> > **Note:** You can use browser devtools to debug your SharedWorker, by entering a URL in your browser address bar to access the devtools workers inspector; for example, in Chrome devtools, the URL [`chrome://inspect/#workers`](chrome://inspect/#workers), and in FireFox, the URL [`about:debugging#workers`](about:debugging#workers).

Reviewer: ```suggestion
> **Note:** You can use browser devtools to debug your SharedWorker, by entering a URL in your browser address bar to access the devtools workers inspector; for example, in Chrome devtools, the URL `chrome://inspect/#workers`, and in FireFox, the URL `about:debugging#workers`.
```
